include/boost/geometry/algorithms/detail/overlay/enrich_intersection_points.hpp

```diff
@@ -285,36 +256,50 @@ struct enriched_map_default_include_policy
 // Add all (non discarded) operations on this ring
 // Blocked operations or uu on clusters (for intersection)
 // should be included, to block potential paths in clusters
-template <typename Turns, typename MappedVector, typename IncludePolicy>
-inline void create_map(Turns const& turns, MappedVector& mapped_vector,
-                       IncludePolicy const& include_policy)
+template <typename Turns, typename IncludePolicy>
+inline auto create_map(Turns const& turns, IncludePolicy const& include_policy)
```

Comment:
This is just a modernization.
(background: next PR needs including discarded here - to verify traversal)